index.js

```diff
@@ -51,6 +51,9 @@ module.exports = {
             RESTServer: require('./lib/network/rest/RESTServer'),
             RESTClient: require('./lib/network/rest/RESTClient'),
         },
+        utils: {
```

Comment:
I think `utils != utils` in this case :)

Here I put it in utils, because it does not do any network but is rather a helper to support network-related operations. Now if you think it's better to put it under the network/ directory anyway, I can change it, let me know.